converters/google/convert.go

```diff
@@ -249,10 +249,15 @@ func (c *Converter) augmentAsset(tfData converter.TerraformResourceData, cfg *co
 // projects. It implements a cache because many resources share the same
 // project.
 func (c *Converter) getAncestry(project string) (string, error) {
+	fmt.Printf("getAncestry: project=%s\n", project)
```

Comment:
Can we hide this behind a verbose flag?

Reply:
yes. Changed to log.Printf which only logs with --verbose.